packages/fast-components-react-base/README.md

```diff
@@ -0,0 +1,50 @@
+# FAST Components React Base
+A set of React components without styles.

```

Comment:
Why would a contributor want to use these Base components?  Sell it here.

Reply:
@janechu I have some things that I've put together we can pull from here if you like.

Reply:
@chrisdholt Great, please let me know what extra information I can put here

Reply:
@janechu  @chrisdholt  looks like this is the only one to not yet be resolved?  Just wanted to confirm so we can get this PR committed.  Thx